### DISCUSSION TITLE
这种字体参数不全的是不是不能添加

### DISCUSSION THREAD

**Discussion by 7peanuts** [F9y4ng/GreasyFork-Scripts#82] [问答专区(Question / Answer)]
【墨黑】悦圆，有四个版本，字体家族名称都没有标全，添加失败，这种是不是就不能添加到脚本里了。
https://www.fontke.com/font/10685274/detail/

**Comment by F9y4ng** [comment]
你提供的这个字体，我连安装都安装不了😅。 这类非官方的修改版字体存在很多兼容问题，多找些类似的字体尝试吧。

**注：这类与脚本程序无关的话题不要发起ISSUES，请在 [Discussions(讨论区)](https://github.com/F9y4ng/GreasyFork-Scripts/discussions) 发帖询问。**

**Comment by 7peanuts** [comment]
> 

哦哦，不好意思 😆 
这四个字体都是ttc格式的，虽然名字一样，但是字重不同，分别是medium、light、ultralight和thin四种，medium和light的可以安装，那俩不行。

**Comment by F9y4ng** [comment]
> 哦哦，不好意思 😆 这四个字体都是ttc格式的，虽然名字一样，但是字重不同，分别是medium、light、ultralight和thin四种，medium和light的可以安装，那俩不行。

**light 字重**  `{"ch":"【墨黑】悦圆","en":"STHeiti-Light"}`

这个字体字型太小了，看得眼疼。😫

**Comment by 7peanuts** [comment]
> > 哦哦，不好意思 😆 这四个字体都是ttc格式的，虽然名字一样，但是字重不同，分别是medium、light、ultralight和thin四种，medium和light的可以安装，那俩不行。
> 
> **light 字重** `{"ch":"【墨黑】悦圆","en":"STHeiti-Light"}`
> 
> 这个字体字型太小了，看得眼疼。😫

感谢，确实非常小，放弃在浏览器上使用了 😱
